migrations/0025-add-public-activity-scope-index.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner, TableIndex } from 'typeorm';
+
+export class AddActivityScopeIndex1608574428247 implements MigrationInterface {
```

Comment:
This is updating an existing index not creating a new one, so does this still work or do we need to drop the original index before creating the new one?

Reply:
When the `public` column was dropped in the previous migration, so was the original index